cli/src/cli.rs

```diff
@@ -241,6 +191,9 @@ pub async fn run(deploy_config: Option<Arc<dyn DeployConfig>>) -> Result<(), Err
             table,
             remote,
         }) => {
+            // Get function names as well as pull all updates from the code.
+            let all_functions = prepare_crates(PathBuf::from(build_config()?.build_path), &crat)?;
```

Comment:
@quesabe let's move this inside of `invoke()` and keep `cli.rs` clean. Same for the `logs` command.

Reply:
Fixed